drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -110,13 +111,16 @@ static struct bcm2835_audio_driver bcm2835_audio_alsa = {
 	.newctl = snd_bcm2835_new_ctl,
 };
 
-static struct bcm2835_audio_driver bcm2835_audio_hdmi0 = {
+/* This struct NOT static as we may wish to reassign the route
```

Comment:
Are you confusing `static` and `const`? static is fine (the structure is only used within this compilation unit/file), but const would be a problem if it were used.

Reply:
Bugger.

Reply:
Fixed. Rookie mistake!